docs/MO_DG/prepare_model/Config_Model_Optimizer.md

```diff
@@ -124,19 +116,15 @@ virtualenv -p /usr/bin/python3.6 .env3/bin/activate
 ```shell
 pip3 install -r requirements.txt
 ```
-> **NOTE**: TensorFlow 1.x and 2.x are incompatible. Use separate virtual environments if you want to install multiple TensorFlow versions.
+> **NOTE**: Since OpenVINO 2022.1 release TensorFlow1 usage is deprecated. TensorFlow 1.x and 2.x are incompatible. Use separate virtual environments if you want to install multiple TensorFlow versions.
 
     *   To install dependencies only for Caffe:
 ```shell
 pip3 install -r requirements_caffe.txt
-```
-    *   To install dependencies only for TensorFlow 1.x:
-```shell
-pip3 install -r requirements_tf.txt
 ```
     *   To install dependencies only for TensorFlow 2.x:
```

Comment:
Remove `only`

Reply:
I don't think only should be removed, it looks ok: dependencies only for caffe, only for mxnet,... Possibly add notice that TF 2 will be installed because tf1.x support deprecated.

Reply:
Ok